[docs] Fix typo on the Joy UI theme builder (#36734)
mui/material-ui: docs/data/joy/customization/theme-builder/theme-builder.md

## Colors

> Use the visual color palette builder below to output a code snippet that you can quickly copy and paste onto your codebase.

Reviewer: It wasn't clear to me exactly where to find the custom code at first glance 😅

```suggestion
Use the visual color palette builder below to construct a custom theme.
Click the **< >** icon in the top right corner when you're ready to copy and paste the code into your app.
```

> For more details about how colors are strucured and work in Joy UI, [read the Colors page](/joy-ui/customization/theme-colors/).

Reviewer: Another typo here, but I'd rephrase this a bit too:

```suggestion
Read [the Theme colors page](/joy-ui/customization/theme-colors/) for details on working with colors in Joy UI.
```
